util/deserialize.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+const { Buffer } = require('buffer')
```

Comment:
This syntax isn't supported by all browsers.

```suggestion
var Buffer = require('buffer').Buffer
```

Same below.